src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_GetDiagnostics.cs

```diff
@@ -281,13 +281,12 @@ protected override async Task AppendDiagnosticsAsync(Project project, IEnumerabl
 
             private bool ShouldIncludeStateSet(Project project, StateSet stateSet)
```

Comment:
nit (for the future).  I've always foudn 'state set' to be vague and unclear.  another name could help :)